Doc/library/os.rst

```diff
@@ -3997,7 +3997,7 @@ Linux extended attributes
 
 .. versionadded:: 3.3
 
-These functions are all available on Linux only.
+These functions are all available on Linux and Cygwin only.
```

Comment:
```suggestion
These functions are all available on Linux only.
```

Reply:
Pushed a new commit that removed the doc changes.